mslice/app/mainwindow.py

```diff
@@ -169,3 +169,15 @@ def show_busy(self, busy):
         else:
             return
         QApplication.processEvents()
+
+
+class ShowBusy():
```

Comment:
I don't think this is the best place for this as it requires the presenter modules to have links back to the mainwindow module.

How about we just have a separate `busy.py` in presenters? We can also do this using [`contextlib`](https://docs.python.org/2/library/contextlib.html) without the need to define a class:

```python
from contextlib import contextmanager

@contextmanager
def show_busy(view):
    self.view.busy.emit(True)
    yield
    self.view.busy.emit(False)
```

then

```python
with show_busy(...):
    ...
```

Reply:
Done, that's way better.